src/sentry/static/sentry/app/components/modals/featureTourModal.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import omit from 'lodash/omit';
+import styled from '@emotion/styled';
+
+import {openModal, ModalRenderProps} from 'app/actionCreators/modal';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import {t} from 'app/locale';
+import {IconClose} from 'app/icons';
+import {callIfFunction} from 'app/utils/callIfFunction';
+import space from 'app/styles/space';
+
+type TourStep = {
+  title: string;
+  image?: React.ReactNode;
```

Comment:
As this is an optional prop, could you please move this prop to the end of the object?